### PR TITLE
fix(tunes): Make label an alias for title in tunes menu item

### PR DESCRIPTION
**Pull request by TatianaFomina**
In order to unify toolbox and block tunes configs formats `label` was renamed to `title` in Tunes Menu config item. Property `label` was left as `title`'s alias for compatibility

### CODE REVIEW

File: src/components/modules/toolbar/blockSettings.ts [hunk 1]
```diff
@@ -192,4 +193,29 @@ export default class BlockSettings extends Module<BlockSettingsNodes> {
   private onOverlayClicked = (): void => {
     this.close();
   };
+
+  /**
+   * Resolves aliases in tunes menu items
+   *
+   * @param item - item with resolved aliases
+   */
+  private resolveAliases(item: TunesMenuConfigItem, aliases: {[alias: string]: string}): TunesMenuConfigItem {
```
Comment (neSpecc) [inline src/components/modules/toolbar/blockSettings.ts]: move to the utils pls

File: src/components/modules/toolbar/blockSettings.ts [hunk 2]
```diff
@@ -192,4 +193,29 @@ export default class BlockSettings extends Module<BlockSettingsNodes> {
   private onOverlayClicked = (): void => {
     this.close();
   };
+
+  /**
+   * Resolves aliases in tunes menu items
```
Comment (neSpecc) [inline src/components/modules/toolbar/blockSettings.ts:199]: it could be more abstract

File: src/components/utils/popover.ts [hunk 3]
```diff
@@ -434,7 +434,7 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
       el.dataset.itemName = item.name;
     }
     const label = Dom.make('div', Popover.CSS.itemLabel, {
-      innerHTML: item.label,
+      innerHTML: item.title,
```
Comment (neSpecc) [inline src/components/utils/popover.ts]: the title is optional, so we need to provide some fallback value

File: src/components/modules/toolbar/blockSettings.ts (hunk 1 above)
Comment (TatianaFomina) [reply src/components/modules/toolbar/blockSettings.ts]: done

File: src/components/utils/popover.ts (hunk 3 above)
Comment (TatianaFomina) [reply src/components/utils/popover.ts]: done

File: src/components/utils/popover.ts [hunk 4]
```diff
@@ -720,4 +720,20 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
...
+   * @param item - item to fill in title of
+   */
+  private fillItemTitle(item: PopoverItem): PopoverItem {
+    if (item.title !== undefined) {
+      return item;
+    }
+
+    return {
+      ...item,
+      title: capitalize(item.name),
```
Comment (neSpecc) [inline src/components/utils/popover.ts]: do not use cypress-name for business logic. We should consider renaming this property to `cyName`

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -1,14 +1,16 @@
 # Changelog
 
-### 2.26.3
+### 2.26.4
 
 - `Fix` — *Paste Module* — fix for a problem with specifying of `pasteConfig().tags` in upper case  [#2208](https://github.com/codex-team/editor.js/issues/2208).
+- `Improvement` — *Menu Config* — Property `label` renamed to `title`.
```
Comment (neSpecc) [inline docs/CHANGELOG.md:5]: leave it under 2.26.3

File: src/components/i18n/locales/en/messages.json [hunk 6]
```diff
@@ -18,7 +18,8 @@
     },
     "popover": {
       "Filter": "",
-      "Nothing found": ""
+      "Nothing found": "",
+      "Untitled": ""
```
Comment (neSpecc) [inline src/components/i18n/locales/en/messages.json]: it makes no sense for end user, lets just leave a button empty

File: src/components/utils/popover.ts [hunk 7]
```diff
@@ -170,22 +176,24 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
...
     filterLabel: string;
     nothingFoundLabel: string;
+    untitledLabel?: string;
     customContent?: HTMLElement;
     customContentFlippableItems?: HTMLElement[];
     scopeElement: HTMLElement;
   }) {
     super();
-    this.items = items;
+    this.items = items.map((item) => this.fillItemTitle(item));
```
Comment (neSpecc) [inline src/components/utils/popover.ts]: seems redundant

File: docs/CHANGELOG.md [hunk 8]
```diff
@@ -1,14 +1,19 @@
 # Changelog
 
-### 2.26.3
+### 2.26.4
 
 - `Fix` — *Paste Module* — fix for a problem with specifying of `pasteConfig().tags` in upper case  [#2208](https://github.com/codex-team/editor.js/issues/2208).
 
+### 2.26.3
+
+- `Improvement` — *Menu Config* — Property `label` renamed to `title`.
```
Comment (neSpecc) [inline docs/CHANGELOG.md]: Pls, swap these two logs

Comment (TatianaFomina) [reply docs/CHANGELOG.md]: 🤦‍♀️

File: src/components/utils/popover.ts [hunk 9]
```diff
@@ -180,6 +181,7 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
     searchable?: boolean;
     filterLabel: string;
     nothingFoundLabel: string;
+    untitledLabel?: string;
```
Comment (neSpecc) [inline src/components/utils/popover.ts]: should be removed?

Comment (TatianaFomina) [reply src/components/utils/popover.ts]: yep

**Review by neSpecc — APPROVED** [review]

File: types/tools/tool-settings.d.ts [hunk 10]
```diff
@@ -28,11 +28,34 @@ export interface ToolboxConfigEntry {
...
+  /**
+   * Tune displayed text.
+   */
+  title?: string;
+
+  /**
+   * Tune displayed text.
+   * Alias for title property
+   */
+  label?: string
```
Comment (ilyamore88) [inline types/tools/tool-settings.d.ts:47]: Maybe mark this field as `@deprecated`?

Comment (TatianaFomina) [reply types/tools/tool-settings.d.ts:47]: good idea

**Review by ilyamore88 — APPROVED** [review]

**Pull request merged**